### ISSUE TITLE
rustdoc: --show-coverage shouldn't count tuple struct fields

### ISSUE THREAD

**Issue by taylordotfish** [rust-lang/rust#83255]
rustdoc's `--show-coverage` flag counts public fields of tuple structs, but this doesn't really make sense, since `doc` attributes on such fields never appear in the generated documentation.

For example, given the following in `src/lib.rs`:

```rust
//! test

/// test
pub struct Test(pub u8);
```

`cargo rustdoc -- -Z unstable-options --show-coverage` reports:

```
+-------------------------------------+------------+------------+------------+------------+
| File                                | Documented | Percentage |   Examples | Percentage |
+-------------------------------------+------------+------------+------------+------------+
| src/lib.rs                          |          2 |      66.7% |          0 |       0.0% |
+-------------------------------------+------------+------------+------------+------------+
| Total                               |          2 |      66.7% |          0 |       0.0% |
+-------------------------------------+------------+------------+------------+------------+
```

If `src/lib.rs` is changed to:

```rust
//! test

/// test
pub struct Test(
    /// test
    pub u8,
);
```

then the coverage is reported as 100%, but, as expected, the documentation for the `pub u8` field isn't included in the documentation generated by `cargo doc`.

### Meta

```
rustc 1.52.0-nightly (f5d8117c3 2021-03-16)
binary: rustc
commit-hash: f5d8117c338a788bd24abec733fd143dfceb25a0
commit-date: 2021-03-16
host: powerpc64le-unknown-linux-gnu
release: 1.52.0-nightly
LLVM version: 12.0.0
```

**Comment by camelid** [comment]
Tuple struct field docs will be rendered soon, assuming that #80320 merges.

**Comment by GuillaumeGomez** [comment]
I guess we can now consider this issue to be fixed since #87451 was merged?

**Comment by camelid** [comment]
@GuillaumeGomez that sounds right to me. Do you want to close it?

**Comment by GuillaumeGomez** [comment]
Let's close it then. :)
